app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def wrapper(*args, **kwargs) -> int:
```

Comment:
Also do not forget to add type hints on `*args and **kwargs` , also you do not every time returning int , it can be any type of data )
As a task says `You have a big array with data`